### ISSUE TITLE
Easiest way to store the token in an https only cookie

### ISSUE THREAD

**Issue by mikepc** [go-oauth2/oauth2#114]
Trying to figure out the easiest way to store the access token and refresh token in an https only cookie.  Have any guidance?

**Comment by exapsy** [comment]
Yeah the `HandleTokenRequest` seems to offer no options to store the token in cookie. If you've found a way @mikepc I would appreciate some guidance :D
